pingserver/README.md

```diff
@@ -105,6 +105,26 @@ Percentage of the requests served within a certain time (ms)
 7,000 concurrent requests:
 
 ```
-Java mean: 36.3 sec
-Go mean:    2.8 sec
+Java median: 36.2 sec
+Go median:    2.3 sec
 ```
+
+### Comparing and plotting many concurrent requests
+compare.py is a script that will compare the response times for a range of concurrent requests, currently set to 1000 to 9000 between Go and Java.
```

Comment:
I'm a little confused about the bounds of the test. The graph shows up to 19,000 while the script explicitly goes up to 20,000. Moreover, the output csv only goes to 19,000.

Reply:
I don't know how to count